packages/tools/src/eventDispatchers/keyboardEventHandlers/keyDown.ts

```diff
@@ -23,7 +23,8 @@ export default function keyDown(evt: KeyDownEventType): void {
   );
 
   const toolName = activeTool.getToolName();
+  const toolInstanceName = activeTool.instanceName;
```

Comment:
Is it necessary to pass both attributes?  The viewports cursor should have all cursors necessary, registered multiple times if needed.